C/include/c4ReplicatorTypes.h

```diff
@@ -259,7 +259,10 @@ typedef struct C4ReplicatorParameters {
                      ///< Implemented by BuiltInWebSocket.
 #define kC4ReplicatorAuthClientCert    "clientCert"     ///< TLS client certificate (value platform-dependent)
 #define kC4ReplicatorAuthClientCertKey "clientCertKey"  ///< Client cert's private key (data)
-#define kC4ReplicatorAuthToken         "token"          ///< Session cookie or auth token (string)
+#define kC4ReplicatorAuthClientCertKeyIsExternal                                                                       \
+    "clientCertKeyIsExternal"           ///< if true key (data) is a pointer to External PrivateKey
```

Comment:
This is a bad idea! You'll have to add a field to C4ReplicatorParameters instead.

Reply:
Okay